### ISSUE TITLE
Discrepancies between lists of torrent properties.

### ISSUE THREAD

**Issue by flipclone** [fedarovich/qbittorrent-cli#16]
When retrieving the torrent list with `torrent list --format json` it returns the following properties for each torrent...

`"hash"
"name"
"magnet_uri"
"size"
"progress"
"dlspeed"
"upspeed"
"priority"
"num_seeds"
"num_complete"
"num_leechs"
"num_incomplete"
"ratio"
"eta"
"state"
"seq_dl"
"f_l_piece_prio"
"category"
"tags"
"super_seeding"
"force_start"
"save_path"
"added_on"
"completion_on"
"tracker"
"dl_limit"
"up_limit"
"downloaded"
"uploaded"
"downloaded_session"
"uploaded_session"
"amount_left"
"completed"
"ratio_limit"
"seen_complete"
"last_activity"
"time_active"
"auto_tmm"
"total_size"
"max_ratio"
"max_seeding_time"
"seeding_time_limit"`

yet when i retrieve properties with `torrent properties <hash> -F json` the only properties i get are...

`"SavePath"
"CreationDate"
"Comment"
"Size"
"TotalWasted"
"TotalUploaded"
"TotalUploadedInSession"
"TotalDownloaded"
"TotalDownloadedInSession"
"UploadLimit"
"DownloadLimit"
"TimeElapsed"
"SeedingTime"
"ConnectionCount"
"ConnectionLimit"
"ShareRatio"
"AdditionDate"
"CompletionDate"
"CreatedBy"
"AverageDownloadSpeed"
"DownloadSpeed"
"AverageUploadSpeed"
"UploadSpeed"
"EstimatedTime"
"LastSeen"
"Peers"
"TotalPeers"
"Seeds"
"TotalSeeds"
"PieceSize"
"OwnedPieces"
"TotalPieces"
"Reannounce"`

therefore i am limited to which properties i can retrieve with `torrent properties <hash> -F property:name=` 

in order to get updates of properties that change, such as "State" "Progress" "eta", i have to retrieve and parse the entire torrent list every time, not a big deal, just kinda slow and tasking.

all in all i was just curious if this was some sort of limitation of the qB webui or just the way qbt-cli was written?

**Comment by fedarovich** [comment]
Yes, it's the limitation of qB web API. Alas, it returns only these properties for the particular torrent.

**Comment by flipclone** [comment]
looking through the Web API docs for V4.1+ there is an option to retrieve "progress" from `get torrent contents`   [https://github.com/qbittorrent/qBittorrent/wiki/Web-API-Documentation#get-torrent-contents](url)  

perhaps this could be worked in?

**Comment by flipclone** [comment]
also looking at documentation shows "eta" under `get torrent generic properties`  [https://github.com/qbittorrent/qBittorrent/wiki/Web-API-Documentation#get-torrent-generic-properties](url)

**Comment by flipclone** [comment]
I see, and im sorry if you got notifications for the other replies i posted and then deleted, i missed "EstimatedTime" hiding in the `torrent properties <hash>` because i was looking for "eta" since that is how it is listed in `torrent list`
